datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -591,10 +559,6 @@ impl LogicalPlan {
     /// // create new plan using rewritten_exprs in same position
     /// let new_plan = plan.new_with_exprs(rewritten_exprs, new_inputs);
     /// ```
-    ///
-    /// Note: sometimes [`Self::with_new_exprs`] will use schema of
-    /// original plan, it will not change the scheam.  Such as
-    /// `Projection/Aggregate/Window`
```

Comment:
This comment looks incorrect. For `Projection/Aggregate/Window`, the schema might change.